recipes/autoconf-archive/all/test_package/configure.ac

```diff
@@ -0,0 +1,11 @@
+AC_INIT([hello], [1.0])
+AC_CONFIG_SRCDIR([hello.c])
+AC_CONFIG_AUX_DIR([autostuff])
+AM_INIT_AUTOMAKE([foreign])
+AC_PROG_CC
+m4_pattern_forbid([^AX_],
```

Comment:
Please use some of the macros available in the autoconf archive.

Reply:
Will try. I'm really just learning how this stuff works. I know next to nothing about autoconf and friends.

Reply:
Great! This is the best way of learning something.

Reply:
Thanks for the fast feedback!